### PR TITLE
Bug severity guidelines docs

### PR DESCRIPTION
**Pull request by amyblais**
- Adding T-5 code freeze deadline
- Adding documentation for bug severity guidelines

### CODE REVIEW

**Comment by amyblais** [comment]
@lindalumitchell @DHaussermann I'm wondering if QAs can take a look and let me know any improvements.

I'll probably move some more checklist items to the new T-5 code freeze deadline over time.

File: source/process/bug-fix-release.md [hunk 1]
```diff
@@ -172,8 +172,18 @@ Review the [Release Features & Bugs Quality Gate Guidelines](https://docs.google
     - Verify all Jira tickets other than newly-filed bugs have been tested, verified, and closed
     - As bug fixes are merged and RCs are cut, verify fixes on new RCs, and post in Release Channel after testing
     - As RCs are cut, update `selenium.test.mattermost.com` to latest RC
+    
+### I. (T-minus 5 working days) Code Freeze
+Day after which only **S1 regressions** should be fixed (crashes, security vulnerabilities) and no other code changes should be comitted. Exceptions can be made by the Release Manager during triage on a case-by-case basis. Review the [Release Features & Bugs Quality Gate Guidelines](https://docs.google.com/document/d/1QxB_A1qkEJBKAvQpRa7JiSQLZhwg6HAEajNRNa7ldGg/edit)
```
Comment (lindalumitchell) [inline source/process/bug-fix-release.md]:
```suggestion
Day after which only **S1 regressions** should be fixed (crashes, security vulnerabilities) and no other code changes should be committed. Exceptions can be made by the Release Manager during triage on a case-by-case basis. Review the [Bug Severity Guidelines](/process/bug-severity-guidelines).
```

File: source/process/bug-fix-release.md [hunk 2]
```diff
@@ -172,8 +172,18 @@ Review the [Release Features & Bugs Quality Gate Guidelines](https://docs.google
...
+
+1. **(Team) Daily Release Update Meeting**
+    - Continue to triage Jira tickets
+    - If no blocking issues are found, PM, Dev, and QA sign off on the release and plan to cut final
```
Comment (lindalumitchell) [inline source/process/bug-fix-release.md]:
```suggestion
    - If no blocking issues are found, PM, Dev, and QA sign off on the release and begin final smoke tests
```

File: source/process/feature-release.md [hunk 3]
```diff
@@ -210,8 +210,18 @@ Day when Leads and PMs decide which major features are included in the release,
     - Verify all Jira tickets other than newly-filed bugs have been tested, verified, and closed
     - As bug fixes are merged and RCs are cut, verify fixes on new RCs, and post in Release Channel after testing
     - As RCs are cut, update `selenium.test.mattermost.com` to latest RC
+    
+### J. (T-minus 5 working days) Code Freeze
+Day after which only **S1 regressions** should be fixed (crashes, security vulnerabilities) and no other code changes should be comitted. Exceptions can be made by the Release Manager during triage on a case-by-case basis. Review the [Release Features & Bugs Quality Gate Guidelines](https://docs.google.com/document/d/1QxB_A1qkEJBKAvQpRa7JiSQLZhwg6HAEajNRNa7ldGg/edit)
```
Comment (lindalumitchell) [inline source/process/feature-release.md]:
```suggestion
Day after which only **S1 regressions** should be fixed (crashes, security vulnerabilities) and no other code changes should be committed. Exceptions can be made by the Release Manager during triage on a case-by-case basis. Review the [Bug Severity Guidelines](/process/bug-severity-guidelines).
```

File: source/process/feature-release.md [hunk 4]
```diff
@@ -210,8 +210,18 @@ Day when Leads and PMs decide which major features are included in the release,
...
+
+1. **(Team) Daily Release Update Meeting**
+    - Continue to triage Jira tickets
+    - If no blocking issues are found, PM, Dev, and QA sign off on the release and plan to cut final
```
Comment (lindalumitchell) [inline source/process/feature-release.md]:
```suggestion
    - If no blocking issues are found, PM, Dev, and QA sign off on the release and begin final smoke tests
```

File: source/process/bug-fix-release.md [hunk 5]
```diff
@@ -168,19 +168,20 @@ Review the [Release Features & Bugs Quality Gate Guidelines](https://docs.google
...
     - Go through all tabs of testing spreadsheet and verify all comments and questions have been filed in Jira as needed
-    - Verify all Jira tickets other than newly-filed bugs have been tested, verified, and closed
-    - As bug fixes are merged and RCs are cut, verify fixes on new RCs, and post in Release Channel after testing
-    - As RCs are cut, update `selenium.test.mattermost.com` to latest RC
     
 ### I. (T-minus 5 working days) Code Freeze
-Day after which only **S1 regressions** should be fixed (crashes, security vulnerabilities) and no other code changes should be comitted. Exceptions can be made by the Release Manager during triage on a case-by-case basis. Review the [Release Features & Bugs Quality Gate Guidelines](https://docs.google.com/document/d/1QxB_A1qkEJBKAvQpRa7JiSQLZhwg6HAEajNRNa7ldGg/edit)
+Day after which only **S1 regressions** should be fixed (crashes, security vulnerabilities) and no other code changes should be committed. Exceptions can be made by the Release Manager during triage on a case-by-case basis. Review the [Bug Severity Guidelines](/process/bug-severity-guidelines).
```
Comment (lindalumitchell) [inline source/process/bug-fix-release.md]: Hm, I realized my suggestion for the link should probably be `(https://docs.mattermost.com/process/bug-severity-guidelines)`. I missed out the beginning of the link. Does that seem right? Sorry about that.

File: source/process/feature-release.md [hunk 6]
```diff
@@ -207,19 +207,20 @@ Day when Leads and PMs decide which major features are included in the release,
...
     - Go through all tabs of testing spreadsheet and verify all comments and questions have been filed in Jira as needed
-    - Verify all Jira tickets other than newly-filed bugs have been tested, verified, and closed
-    - As bug fixes are merged and RCs are cut, verify fixes on new RCs, and post in Release Channel after testing
-    - As RCs are cut, update `selenium.test.mattermost.com` to latest RC
     
 ### J. (T-minus 5 working days) Code Freeze
-Day after which only **S1 regressions** should be fixed (crashes, security vulnerabilities) and no other code changes should be comitted. Exceptions can be made by the Release Manager during triage on a case-by-case basis. Review the [Release Features & Bugs Quality Gate Guidelines](https://docs.google.com/document/d/1QxB_A1qkEJBKAvQpRa7JiSQLZhwg6HAEajNRNa7ldGg/edit)
+Day after which only **S1 regressions** should be fixed (crashes, security vulnerabilities) and no other code changes should be committed. Exceptions can be made by the Release Manager during triage on a case-by-case basis. Review the [Bug Severity Guidelines](/process/bug-severity-guidelines).
```
Comment (lindalumitchell) [inline source/process/feature-release.md]: Same comment here:

Hm, I realized my suggestion for the link should probably be `(https://docs.mattermost.com/process/bug-severity-guidelines)`. I missed out the beginning of the link. Does that seem right? Sorry about that.

**Pull request merged**